push_docker.sh

```diff
@@ -1,42 +1,42 @@
+#! /bin/bash 
+
 DOCKER_IMAGE_FILE=$1
 REGISTRY_SERVER=$2
-REGISTRY_PORT=$3
-REGISTRY_USERNAME=$4
-REGISTRY_PASSWD=$5
-DOCKER_IMAGE_TAG=$6
+REGISTRY_USERNAME=$3
+REGISTRY_PASSWD=$4
+PLATFORM=$5
+SONIC_VERSION=$6
+DOCKER_IMAGE_TAG=$7
+
+push_it() {
```

Comment:
>push_it [](start = 0, length = 7)

Could you add code comments above to explain the parameters?

Reply:
ok

Reply:
Done.